### DISCUSSION TITLE
Missing 'Specials' for Attack on Titan?

### DISCUSSION THREAD

**Discussion by bverkron** [sct/overseerr#3684] [Help]
Just noticed that the final few episodes of Attack on Titan (which are considered Specials and not apart of the normal seasons) are not available in Overseer. IIRC other shows that have special episodes show up in Overseer and can be requested, but not with Attack on Titan. Am I remember wrong?

![Screenshot 2023-11-09 at 7 10 24 PM](https://github.com/sct/overseerr/assets/19174503/c256eca9-0eee-4ae6-9c5f-711110275d7c)

![Screenshot 2023-11-09 at 7 09 37 PM](https://github.com/sct/overseerr/assets/19174503/83e2d86c-770d-4f9f-9277-f2c1e772ac77)

![Screenshot 2023-11-09 at 7 11 47 PM](https://github.com/sct/overseerr/assets/19174503/3b2128f5-67d0-4799-b381-b380e2d404fc)

**Comment by bverkron** [comment]
Just found this issue. Guess I was remembering wrong? Unfortunate.

https://github.com/sct/overseerr/discussions/3684

**Reply by AhmedNSidd** [reply]
Just updating this thread to say that I'm looking into this issue. I have a solution that seems to be working locally... I should have a PR out for this soon. Sometime this week I hope.

**Reply by bverkron** [reply]
That's amazing, thank you! Out of curiosity what was the issue?

**Reply by AhmedNSidd** [reply]
The issue was that it seems like Specials was explicitly being blocked out using if-conditions.

Your guess was right. Overseerr IS getting Specials (or "Season 0") through its APIs by contacting TMDB, but it was being blocked out essentially.

All I had to do was basically remove that if-condition.

Now Specials is being treated just as another Season in a TV show, but it's just labelled as "Specials".

**Reply by AhmedNSidd** [reply]
PR is out! https://github.com/sct/overseerr/pull/3724

Happy Holidays :)
